bin/vyper

```diff
@@ -37,9 +33,16 @@ parser.add_argument('input_files', help='Vyper sourcecode to compile', nargs='+'
 parser.add_argument('--version', action='version', version='{0}'.format(vyper.__version__))
 parser.add_argument('--show-gas-estimates', help='Show gas estimates in ir output mode.', action="store_true")
 parser.add_argument('-f', help=format_options_help, default='bytecode', dest='format')
+parser.add_argument('--debug', help='Add compiler debugging output', action='store_true')
 
 args = parser.parse_args()
 
+tb_limit = os.environ.get('VYPER_TRACEBACK_LIMIT')
+if tb_limit:
+    sys.tracebacklimit = int(tb_limit)
+if not args.debug:
+    sys.tracebacklimit = 0
+
```

Comment:
Only check if flag is provided?

```suggestion
sys.tracebacklimit = 0
if args.debug:
    tb_limit = os.environ.get('VYPER_TRACEBACK_LIMIT')
    if tb_limit:
        sys.tracebacklimit = int(tb_limit)
```

Reply:
I re-did the commit, the automatic commit was somehow messed up.

Reply:
Actually, isn't `int(None) =0`? Can get rid of that inner if statement too?